src/modules/internal_resistance/internal_resistance_params.c

```diff
@@ -0,0 +1,2 @@
+PARAM_DEFINE_INT32(INTER_RES_EN, 1);
+PARAM_DEFINE_FLOAT(RIN_UPDATE_TIME, 30.0);
```

Comment:
```suggestion
PARAM_DEFINE_FLOAT(RIN_UPDATE_TIME, 30.0);
```

Name all the related parameters with the same prefix so they're grouped together. Probably `RIN`?